### PR TITLE
feat: Oracle source

### PR DESCRIPTION
**Pull request by TCeason**
Support Oracle source

### CODE REVIEW

File: cmd/main_test.go [hunk 1]
```diff
@@ -221,6 +392,58 @@ func prepareMysql() {
...
+			varchar_col VARCHAR2(255),
+			float_col NUMBER,
+			bool_col NUMBER(1) CHECK (bool_col IN (0, 1)),
+			de NUMBER(18,6),
+			date_col DATE,
+			datetime_col TIMESTAMP,
+			timestamp_col TIMESTAMP
+		)
+	`)
+	// need to test the TIME type in mysql
```
Comment (hantmac) [inline cmd/main_test.go:429]: `// need to test the TIME type in mysql` this comment can be removed here :)

File: cmd/main_test.go [hunk 2]
```diff
@@ -259,6 +498,33 @@ func prepareTestConfig() *cfg.Config {
 		SourceQuery:          "select * from mydb.test_table",
 		SourceSplitKey:       "id",
 		SourceSplitTimeKey:   "",
+		DatabendDSN:          "https://dbarchiver:abc123@tn3ftqihs--medium-p8at.gw.aws-us-east-2.default.databend.com:443",
```
Comment (hantmac) [inline cmd/main_test.go:501]: Be care of the `DatabendDSN`, maybe change it to `http://databend:databend@localhost:8080` is better

Comment (TCeason) [reply cmd/main_test.go]: My mistake. Forget to remove mysql config.

File: cmd/main_test.go (hunk 1 above)
Comment (TCeason) [reply cmd/main_test.go]: Done

**Comment by TCeason** [comment]
ci passed. @hantmac

**Review by hantmac — APPROVED** [review]
LGTM!

**Comment by hantmac** [comment]
> ci passed. @hantmac

@TCeason Great! LGTM! And how about add the docs about syncing oracle?

**Pull request merged**